apps/condo/pages/onboarding.tsx

```diff
@@ -45,8 +45,10 @@ const OnBoardingPage: IOnBoardingIndexPage = () => {
     }, [refetchOnBoarding])
 
     useEffect(() => {
-        if (get(onBoarding, 'completed', false)) {
-            router.push('/')
+        // In some cases user may not have a connected `OnBoarding` record, like after invite with creating new `User` in `InviteNewOrganizationEmployeeService`
+        // So, when no `OnBoarding` record is connected, redirect from this page
+        if (!onBoarding || onBoarding.completed) {
```

Comment:
During the first rendering `onBoarding` will be null. What about checking the `isLoading` variable?

Reply:
Fixed